src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/NonMatchingEidasAcceptanceTest.java

```diff
@@ -160,11 +179,9 @@ public void shouldMapAttributesCorrectly() throws Exception {
         dateOfBirth.getAttributeValues().add(dateOfBirthValue);
         attributeStatementBuilder.addAttribute(dateOfBirth);
 
-        org.opensaml.saml.saml2.core.Response samlResponse = aValidEidasResponse(
-                "requestId",
-                appWithEidasEnabled.getCountryEntityId(),
-                attributeStatementBuilder.build())
-                .build();
+        org.opensaml.saml.saml2.core.Response samlResponse =
+            aValidEidasResponseSignedByStubCountryWithAnAssertionSignedByTheCountry("requestId", attributeStatementBuilder)
```

Comment:
Not sure how I feel about those super long builder method names...

Reply:
I have a plan to fix this.